taxonomy-editor-frontend/src/pages/project/editentry/index.tsx

```diff
@@ -45,13 +62,22 @@ const EditEntry = ({ taxonomyName, branchName, id }: EditEntryProps) => {
           <Typography sx={{ mb: 2, mt: 2, ml: 2 }} variant="h4">
             You are now editing &quot;{id}&quot;
```

Comment:
The text should depend on if the node is editable

Reply:
you can still "edit" by adding children to external nodes 🥸